pages/pipelines/branch_configuration.md.erb

```diff
@@ -30,7 +28,7 @@ steps:
     command:
       - "npm install"
       - "tests.sh"
-    branches: "feature/* release/* master !infrastructure"
+    branches: "master feature/* release/*"
```

Comment:
I'm slightly confused by saying you can do both and removing the 'not' from the first example.

Reply:
I removed the negative because it was redundant, it already did not match the other patterns.

But perhaps we would modify this pipeline yaml to include a meaningful example?

Reply:
```suggestion
    branches: "master feature/* !feature/beta release/*"
```